diff --git a/config.def.h b/config.def.h
index b04d224..91ff982 100644
--- a/config.def.h
+++ b/config.def.h
@@ -160,8 +160,8 @@ static unsigned int defaultattr = 11;
  */
 static MouseShortcut mshortcuts[] = {
 	/* button               mask            string */
-	{ Button4,              XK_NO_MOD,      "\031" },
-	{ Button5,              XK_NO_MOD,      "\005" },
+	// { Button4,              XK_NO_MOD,      "\031" },
+	// { Button5,              XK_NO_MOD,      "\005" },
 };
 
 MouseKey mkeys[] = {
